### ISSUE TITLE
config parsing in send_gearman produces errors in the output

### ISSUE THREAD

**Issue by jstark1** [sni/mod_gearman#162]
we use send_gearman to submit passive checkresults to gearmand, like this:
```
/omd/versions/default/bin/send_gearman --config=$HOME/etc/mod-gearman/worker.cfg --server=XXX --host=XXX --service='XXX' --message="XXXX" --returncode=X
```

its only cosmetic and the command successes, but on the console are errors printed like this:
```
 [2022-10-13 10:51:03][20353][ERROR] unknown option 'sink-rate'
 [2022-10-13 10:51:03][20353][ERROR] unknown option 'backgrounding-threshold'
 [2022-10-13 10:51:03][20353][ERROR] unknown option 'mem_limit'
```

**Comment by sni** [comment]
oh right... For now you could use a separate config file without those options.

**Comment by sni** [comment]
fixed with ad9774e37794c12059d0b5815477aa983cb8df50
